components/my-account/Lists/ListTable/ListTable.tsx

```diff
@@ -0,0 +1,171 @@
+  isLoading: boolean
+}
+
+const ListTable = (props: ListTableProps) => {
+  const { rows, onDeleteList, onCopyList, onEditList, isLoading } = props
+  // state for mobile menu
+  const [anchorEl, setAnchorEL] = useState<HTMLElement | null>(null)
+
+  const { t } = useTranslation('common')
+  const theme = useTheme()
+  const mdScreen = useMediaQuery(theme.breakpoints.up('md'))
+
+  const options = [
+    { name: 'Edit', onClick: onEditList },
+    { name: 'Add list items to cart', onClick: () => alert('Work in progress') },
+    { name: 'Initiate Quote', onClick: () => alert('Work in progress') },
+    { name: 'Duplicate', onClick: onCopyList },
+    { name: 'Delete', onClick: onDeleteList },
+  ]
+
+  return (
+    <TableContainer sx={{ opacity: isLoading ? '0.5' : '1' }}>
+      <Table style={{ tableLayout: 'fixed' }}>
+        <TableHead>
+          <TableRow style={{ backgroundColor: '#f7f7f7', padding: '10px 0' }}>
+            <TableCell style={{ padding: '10px 10px', width: mdScreen ? '25%' : '50%' }}>
+              {t('list-name')}
+            </TableCell>
+            <TableCell style={{ padding: '10px 10px', width: mdScreen ? '15%' : '30%' }}>
+              {t('date-created')}
+            </TableCell>
+            {mdScreen && (
+              <TableCell style={{ padding: '10px 10px', width: '20%' }}>
+                {t('created-by')}
+              </TableCell>
+            )}
+            <TableCell
+              style={{ padding: '10px 10px', width: mdScreen ? '25%' : '10%' }}
+            ></TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {rows.map((item: CrWishlist) => {
+            return (
+              <TableRow key={item.id}>
+                <TableCell style={{ ...styles.tableCellStyles, width: mdScreen ? '25%' : '50%' }}>
+                  {mdScreen ? (
+                    item.name
+                  ) : (
+                    <Box>
+                      {item.name}
+                      <br />
+                      <p style={{ margin: '5px 0', color: '#cdcdcd' }}>
+                        {item.auditInfo?.createBy}
+                      </p>
+                    </Box>
+                  )}
+                </TableCell>
+                <TableCell style={{ ...styles.tableCellStyles, width: mdScreen ? '15%' : '30%' }}>
+                  {formatDate(item.auditInfo?.createDate)}
+                </TableCell>
+                {mdScreen && (
+                  <TableCell style={{ ...styles.tableCellStyles, width: '20%' }}>
+                    {item.auditInfo?.createBy}
+                  </TableCell>
+                )}
+                <TableCell style={{ ...styles.tableCellStyles, width: mdScreen ? '25%' : '10%' }}>
+                  {mdScreen ? (
+                    <Box style={{ justifyContent: 'flex-end', display: 'flex' }}>
+                      <Button sx={styles.iconButtonStyles} data-testid="initiateQuoteBtn">
```

Comment:
Remove button styling. Use variant and color

Reply:
done